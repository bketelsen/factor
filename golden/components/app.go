package components

type App struct {
	//factor.Router
}
