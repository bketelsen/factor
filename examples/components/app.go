package components

import (
	"github.com/factorapp/factor/examples/models"
)

type App struct {
	//factor.Router
	Todos []*models.Todo
	Page  string
}
