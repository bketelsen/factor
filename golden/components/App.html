<main>
    <components:Nav MyProp="Red"/>
    <div>Brian Was Here</div>
</main>
